### ISSUE TITLE
How to publish docs

### ISSUE THREAD

**Issue by andrewbaxter** [hashicorp/terraform-plugin-docs#201]
This is missing from the website, not sure where to reach out.

AFAICT the way to publish docs is
- Docs are in `/docs` at top level of repo
- Docs are committed
- The docs will be shown in the registry for the git version associated with a release

Is this right? Is there a way to publish the docs without committing them (avoiding committing generated files)?

In any case though it would be good to clarify this in the documentation.

**Comment by jacobbednarz** [comment]
yes, that is the general flow. the [hashicorp docs](https://developer.hashicorp.com/terraform/registry/providers/docs) have a high level overview of this too. you need to commit the files for them to be present in the registry once the provider triggers the build.
